### DISCUSSION TITLE
vSphere/ESXi acceptable versions?

### DISCUSSION THREAD

**Discussion by kmcgrego1307** [vmware/packer-examples-for-vsphere#159] [Help] [answered]
Hi, how much of this build system requires ESXi 7U2? We have 6.7U2; will a lot of stuff fail, or would it mostly work?

**Comment by tenthirtyam** [comment, marked as answer]
You would need to modify the configuration (in the common vars) to opt-out of the default configuration to publish the images to the content library. 

All images in this project use EFI and/or EFI with Secure Boot for the guest operating systems. As such, the virtual machines **will not** deploy correctly from the content library as the firmware will flip back to BIOS. This was addressed in 7.0 U2.

Please be mindful of the end of general support for vSphere 6.7 later in 2022.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.
